timbermill-java/timbermill-server/src/main/java/com/datorama/timbermill/server/service/GzippedInputStreamWrapper.java

```diff
@@ -0,0 +1,161 @@
+package com.datorama.timbermill.server.service;
```

Comment:
https://stackoverflow.com/questions/16638345/how-to-decode-gzip-compressed-request-body-in-spring-mvc